Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -1148,6 +1142,12 @@ extension ClusterSystem {
             )
 
             do {
+                guard let actor = self.resolve(id: recipient) else {
+                    self.log.error("Unable to resolve recipient \(recipient). Message will be dropped: \(invocation)")
```

Comment:
Only sent it to dead letters, no need to log here -- dead letters will do the logging 👍 
```suggestion

```